dev/traffic_router/conf/traffic_ops.properties

```diff
@@ -0,0 +1,13 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+traffic_ops.username=admin
+traffic_ops.password=twelve12
```

Comment:
Actually, IMO neither this nor CiaB should use `twelve12`, since that's not accepted by the API either. The password used here should ideally match the current CiaB for ease of use, but CiaB should not contain data we know to be invalid. If you are using the development or testing/demo setups in production, you have ignored all instructions to the contrary and in CiaB's case multiple warnings. We cannot reasonably prevent someone so determined to do improper things from doing those things.

Reply:
> The password used here should ideally match the current CiaB for ease of use, but CiaB should not contain data we know to be invalid.

Okay, PRed making the password `twelve` in both places in #7149